project-templates/staff/leagues/league_division_addteam.html

```diff
@@ -0,0 +1,45 @@
+{% extends 'staff/staffbase.html' %}
+
+{% load static %}
+<link rel="stylesheet" href="{% static "css/staff.css" %}">
+{% block title %}
+    Add Team to Division #{{ division.id }}
+{% endblock %}
+{% block body %}
+    <link rel="stylesheet" href="{% static "css/staff.css" %}">
+    <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+    <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.22.2/moment.min.js"></script>
+    <script type="text/javascript"
+            src="https://cdnjs.cloudflare.com/ajax/libs/tempusdominus-bootstrap-4/5.0.1/js/tempusdominus-bootstrap-4.min.js"></script>
+    <link rel="stylesheet"
+          href="https://cdnjs.cloudflare.com/ajax/libs/tempusdominus-bootstrap-4/5.0.1/css/tempusdominus-bootstrap-4.min.css"/>
+
+    {% if form.errors %}
+        {% for field in form %}
+            {% for error in field.errors %}
+                <div class="alert alert-danger">
+                    <strong>{{ error|escape }}</strong>
+                </div>
+            {% endfor %}
+        {% endfor %}
+        {% for error in form.non_field_errors %}
+            <div class="alert alert-danger">
+                <strong>{{ error|escape }}</strong>
+            </div>
+        {% endfor %}
+    {% endif %}
+    <h1><a href="{% url 'staff:list_league' %}">Return to league list</a></h1>
+
+    <form enctype="multipart/form-data" method="post">
+        {% csrf_token %}
+        {{ field.errors }}
+
+        <label>Team id</label>
+        {{ form.teamid }}

```

Comment:
Make this more user friendly

Reply:
how? in a later issue potentially, I don't see anything wrong with this.